test/integration/visitor_can_add_items_to_cart_test.rb

```diff
@@ -80,11 +80,11 @@ class VisitorCanAddItemsToCartTest < ActionDispatch::IntegrationTest
     category_2 = Oil.create(name: "Coconut Oil")
 
     item_1 = Chip.create(name: "Slotachips", price: 20, description: "Super yummy", oil_id: category_1.id)
-    item_2 = Chip.create(name: "Trader Joe's BBQ", price: 15, description: "I'd trade slota for these!", oil_id: category_2.id)
+    item_2 = Chip.create(name: "Trader Joe's BBQ", price: 15, description: "I'  d trade slota for these!", oil_id: category_2.id)
```

Comment:
Useless assignment to variable - `item_2`. Did you mean `item_1`?<br>Line is too long. [129/80]